pkg/adaptor/cloud/aws/types.go

```diff
@@ -17,7 +17,11 @@ func (i *securityGroupIds) String() string {
 }
 
 func (i *securityGroupIds) Set(value string) error {
-	*i = append(*i, strings.Split(value, ",")...)
+	if len(value) == 0 {
+		*i = make(securityGroupIds, 0)
```

Comment:
is it needed? AFAIR append works on nil slice

Reply:
Hmm.. I'll recheck